package.json

```diff
@@ -21,28 +22,28 @@
     }
   ],
   "dependencies": {
-    "prosemirror-keymap": "^1.0.0",
-    "prosemirror-model": "^1.0.0",
-    "prosemirror-state": "^1.0.0",
-    "prosemirror-transform": "^1.0.0",
-    "prosemirror-view": "^1.0.0"
+    "prosemirror-keymap": "^1.1.2",
```

Comment:
any reason why you bumped our dependencies?

Reply:
Initially done with npm install and audit fix. I kept the changes since I thought it would make clear that at least these minimum versions are required for compatibility with  the recent ESM support improvements.

Reply:
mmm tbh, I dont know what was the original decision to keep it on `1.0.0` version.

@eshvedai any thoughts? Maybe it was done like this to not force consumers to have a minimum PM version. Or might be to prevent `prosemirror` duplicates, no?